src/core_modules/capture-core/components/WidgetsRelationship/common/hooks/useLinkedEntityGroups.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import { useCallback, useEffect, useState } from 'react';
+import log from 'loglevel';
+import moment from 'moment';
+import { errorCreator } from 'capture-core-utils';
```

Comment:
Can we use deps from our internal repo?

Reply:
`errorCreator` is already used by other self-contained widgets. It looks like an easy function to re-rewrite, so maybe we can move it inside each widget?